embassy-stm32/src/rcc/wl.rs

```diff
@@ -1,4 +1,6 @@
-use crate::pac::{FLASH, RCC};
+use stm32_metapac::pwr::vals::Dbp;
```

Comment:
`use crate::pac::...` for consistency with the others

Reply:
Fixed!